tests/js/core.webaudio.js

```diff
@@ -1,259 +1,272 @@
+import { Howl, Howler } from '../../dist/core.js';
+
 // Cache the label for later use.
-var label = document.getElementById('label');
-var start = document.getElementById('start');
+const label = document.getElementById('label');
+const start = document.getElementById('start');
 
 // Setup the sounds to be used.
-var sound1 = new Howl({
-  src: ['audio/sound1.webm', 'audio/sound1.mp3']
+const sound1 = new Howl({
+  src: ['audio/sound1.webm', 'audio/sound1.mp3'],
 });
 
-var sound2 = new Howl({
+const sound2 = new Howl({
   src: ['audio/sound2.webm', 'audio/sound2.mp3'],
   sprite: {
     one: [0, 450],
     two: [2000, 250],
     three: [4000, 350],
     four: [6000, 380],
     five: [8000, 340],
-    beat: [10000, 11163]
-  }
+    beat: [10000, 11163],
+  },
 });
 
 // Enable the start button when the sounds have loaded.
-sound1.once('load', function() {
+sound1.once('load', () => {
   start.removeAttribute('disabled');
   start.innerHTML = 'BEGIN CORE TESTS';
 });
 
 // Define the tests to run.
-var id;
-var tests = [
-  function(fn) {
-    sound1.once('play', function() {
+let id;
+const tests = [
+  (fn) => {
+    sound1.once('play', () => {
       label.innerHTML = 'PLAYING';
       setTimeout(fn, 2000);
     });
-    
+
     id = sound1.play();
   },
 
-  function(fn) {
+  (fn) => {
     sound1.pause(id);
 
     label.innerHTML = 'PAUSED';
     setTimeout(fn, 1500);
   },
 
-  function(fn) {
+  (fn) => {
     sound1.play(id);
 
     label.innerHTML = 'RESUMING';
     setTimeout(fn, 2000);
   },
 
-  function(fn) {
+  (fn) => {
     sound1.stop(id);
 
     label.innerHTML = 'STOPPED';
     setTimeout(fn, 1500);
   },
 
-  function(fn) {
+  (fn) => {
     sound1.play(id);
 
     label.innerHTML = 'PLAY FROM START';
     setTimeout(fn, 2000);
   },
 
-  function(fn) {
+  (fn) => {
     sound1.rate(1.5, id);
 
     label.innerHTML = 'SPEED UP';
     setTimeout(fn, 2000);
   },
 
-  function(fn) {
+  (fn) => {
     sound1.rate(1, id);
 
     label.innerHTML = 'SLOW DOWN';
     setTimeout(fn, 2000);
   },
 
-  function(fn) {
+  (fn) => {
     sound1.fade(1, 0, 2000, id);
 
     label.innerHTML = 'FADE OUT';
-    sound1.once('fade', function() {
-      fn();
-    }, id);
+    sound1.once(
+      'fade',
+      () => {
+        fn();
+      },
+      id,
+    );
   },
 
-  function(fn) {
+  (fn) => {
     sound1.fade(0, 1, 2000, id);
 
     label.innerHTML = 'FADE IN';
-    sound1.once('fade', function() {
-      fn();
-    }, id);
+    sound1.once(
+      'fade',
+      () => {
+        fn();
+      },
+      id,
+    );
   },
 
-  function(fn) {
+  (fn) => {
     sound1.mute(true, id);
 
     label.innerHTML = 'MUTE';
     setTimeout(fn, 1500);
   },
 
```

Comment:
Muting with `Web Audio API` doesn't work in Firefox 93 for windows.

Interestingly enough, the `2.x` version of the library + tests doesn't work in Firefox either!

In Chrome 94, both the `2.x` and `3.x` versions work as expected.

Could this be a general bug that is related to the implementation details, rather than the refactoring?